scripts/cmake/CompilerSetup.cmake

```diff
@@ -148,3 +160,4 @@ endif()
 # preceding cxx flags definition in order to override earlier flags, e.g. for
 # optimization.
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OGS_CXX_FLAGS}")
+add_compile_options(${OGS_CXX_FLAGS})
```

Comment:
Just curious, why is this `add_compile_options` needed beside the setting of the `CMAKE_CXX_FLAGS`?